modules/crypto/src/main/resources/decryption-keys.properties

```diff
@@ -0,0 +1 @@
+V001=<insert decryption key here during deployment>
```

Comment:
can the key be derived as well? if some using this project, wanna use their style of keys it's not possible